test/Feature/HLSLLib/rcp.32.test

```diff
@@ -0,0 +1,67 @@
+#--- source.hlsl
+
+StructuredBuffer<float4> In : register(t0);
+
+RWStructuredBuffer<float4> Out : register(u1);
+
+[numthreads(1,1,1)]
+void main() {
+  Out[0] = rcp(In[0]);
+  float4 Tmp = {rcp(In[1].xyz), rcp(In[1].w)};
+  Out[1] = Tmp;
+  float4 Tmp2 = {rcp(In[2].xy), rcp(In[2].zw)};
+  Out[2] = Tmp2;
+}
+
+
+//--- pipeline.yaml
+
+---
+Shaders:
+  - Stage: Compute
+    Entry: main
+    DispatchSize: [1, 1, 1]
+Buffers:
+  - Name: In
+    Format: Float32
+    Stride: 16
+    Data: [ nan, -inf, -0x1.e7d42cp-127, -0, 0, 0x1.e7d42cp-127, inf, 1, -1, 2.5, -0.25, 2]
+    #  NaN, -Inf, -denorm, -0, 0, denorm, Inf, 1, -1, 2.5, -0.25, 2
+  - Name: Out
+    Format: Float32
+    Stride: 16
+    ZeroInitSize: 48
+  - Name: ExpectedOut # The result we expect
+    Format: Float32
+    Stride: 16
+    Data: [ nan, -0, -inf, -inf, inf, inf, 0, 1, -1, 0.4, -4 , 0.5]
+    #  NaN, NaN, Inf, Inf, Inf, Inf, NaN, 1, -1, 0.4, -4, 0.5
```

Comment:
```suggestion
    Data: [ nan, -0, -inf, -inf, inf, inf, 0, 1, -1, 0.4, -4, 0.5]
    #  NaN, -0, -Inf, -Inf, Inf, Inf, 0, 1, -1, 0.4, -4, 0.5
```
Or remove the comment # since it doesn't seem necessary here for readability.